browse/src/main/kotlin/com/alfresco/content/browse/BrowseViewState.kt

```diff
@@ -12,6 +12,7 @@ import com.alfresco.list.merge
 import com.alfresco.list.replace
 import java.time.ZonedDateTime
 import java.time.temporal.ChronoField
+import kotlinx.coroutines.flow.merge
```

Comment:
This is an unnecessary import the `merge` reference is `com.alfresco.list.merge`